packages/stark-core/src/util/http.util.ts

```diff
@@ -5,19 +5,21 @@ import { convertMapIntoObject } from "./util-helpers";
 import reduce from "lodash-es/reduce";
 
 /**
- * A custom implementation of HttpParameterCodec to correctly encode query parameters.
- * @link https://github.com/NationalBankBelgium/stark/issues/1130
+ * An instance of the {@link StarkHttpParameterCodec} which is internally used by
+ * the [StarkHttpUtil convertStarkQueryParamsIntoHttpParams]{@link StarkHttpUtil#convertStarkQueryParamsIntoHttpParams} method.
+ *
+ * See {@link https://github.com/NationalBankBelgium/stark/issues/1130}
  */
 export const STARK_HTTP_PARAM_ENCODER: HttpParameterCodec = new StarkHttpParameterCodec();
 
 /**
- * Util class used for the HTTP module
- * @dynamic See: https://angular.io/guide/aot-compiler#strictmetadataemit
+ * Util class used for the {@link StarkHttpModule}
+ * @dynamic See: https://v7.angular.io/guide/aot-compiler#strictmetadataemit
```

Comment:
In fact, the `@dynamic` tag was added in this PR: https://github.com/NationalBankBelgium/stark/pull/1360

So I double checked and I managed to add a workaround in some cases in order to remove the tag from some classes (`stark-core/src/util/http.util.ts` and `stark-core/src/modules/xsrf/services/xsrf.service.ts`). However we still need the tag in the other classes (`stark-core/src/configuration/entities/application/app-config.entity.ts` and `stark-core/src/configuration/entities/language/language.entity.ts`) but now I added a comment to explain why is needed 👍.

And regarding compodoc, it doesn't care about these tags which is fine 😉

Reply:
In fact I also managed to add a workaround for the other two classes: `stark-core/src/configuration/entities/application/app-config.entity.ts` and `stark-core/src/configuration/entities/language/language.entity.ts`.

It turns out that using simple strings instead of regex strings solved the issue 😉 

So we don't have any `@dynamic` tag in our code anymore 💪